docs/source/export.rst

```diff
@@ -1,11 +1,293 @@
+    """
+
+Some additional things to note:
+
+* Through the :func:``torch.export.dynamic_dim`` API, we specified the first
+  dimension of each input to be dynamic. Looking at the inputs ``arg4_1`` and
+  ``arg5_1``, they have a symbolic shape of [s0, 64] and [s1, 128], instead of
+  the [32, 64] and [32, 128] shaped tensors that we passed in as example inputs.
+  ``s0`` and ``s1`` are symbols representing that this dimension can be a range
+  of values.
+
+* ``exported_program.range_constraints`` describes the ranges of each symbol
+  appearing in the graph. In this case, we see that ``s0`` and ``s1`` have
+  the range [2, inf]. For technical reasons that are difficult to explain here,
+  they are assumed to be not 0 or 1. This is not a bug, and does not necessarily
+  mean that the exported program will not work for dimensions 0 or 1. See
+  `The 0/1 Specialization Problem <https://docs.google.com/document/d/16VPOa3d-Liikf48teAOmxLc92rgvJdfosIy-yoT38Io/edit?fbclid=IwAR3HNwmmexcitV0pbZm_x1a4ykdXZ9th_eJWK-3hBtVgKnrkmemz6Pm5jRQ#heading=h.ez923tomjvyk>`_
+  for an in-depth discussion of this topic.
+
+* ``exported_program.equality_constraints`` describes which dimensions are
+  required to be equal. Since we specified in the constraints that the first
+  dimension of each argument is equivalent,
+  (``dynamic_dim(example_args[0], 0) == dynamic_dim(example_args[1], 0)``),
+  we see in the equality constraints the tuple specifying that ``arg4_1``
+  dimension 0 and ``arg5_1`` dimension 0 are equal.
+
+
+Limitations of torch.export
+---------------------------
+
+Graph Breaks
+^^^^^^^^^^^^
+
+As ``torch.export`` is a one-shot process for capturing a computation graph from
+a PyTorch program, it might ultimately run into untraceable parts of programs as
+it is nearly impossible to support tracing all PyTorch and Python features. In
+the case of ``torch.compile``, an unsupported operation will cause a "graph
+break" and the unsupported operation will be run with default Python evaluation.
+In contrast, ``torch.export`` will require users to provide additional
+information or rewrite parts of their code to make it traceable. As the
+tracing is based on TorchDynamo, which evaluates at the Python
+bytecode level, there will be significantly fewer rewrites required compared to
+previous tracing frameworks.
+
+When a graph break is encountered, :ref:`ExportDB <torch.export_db>` is a great
+resource for learning about the kinds of programs that are supported and
+unsupported, along with ways to rewrite programs to make them traceable.
+
+Control Flow
+^^^^^^^^^^^^
+
+Graph breaks can also be encountered on general control flow
+(ex. ``if x[0] > 4``), which a tracing compiler cannot possibly deal with
+without generating code for a combinatorially exploding number of paths. In such
+cases, users will need to rewrite their code using special
+**Control Flow Operators**.
```

Comment:
I think it would still be good to mention them, but I won't add any examples and I'll add a "coming soon!" message. Similarly, the constrain APIs are not exactly finalized right? Because we don't want to keep them in export?

Reply:
Good idea, just mention that we intend to support it and it would be available soon.